src/Console/ConfigurationResolver.php

```diff
@@ -493,6 +498,14 @@ public function shouldStopOnViolation()
     }
 
     /**
+     * @return bool
+     */
+    public function configFinderIsOverridden()
+    {
+        return $this->configFinderOverridden;
```

Comment:
if one wont call `getFinder` first, this will wrongly be always `false`

Reply:
Test added.